packages/testing/src/execution_testing/cli/pytest_commands/plugins/execute/rpc/hive.py

```diff
@@ -235,6 +235,12 @@ def test_suite_description() -> str:
     return "Execute EEST tests using hive endpoint."
 
 
+@pytest.fixture(scope="function")
+def test_case_description(request: pytest.FixtureRequest) -> str:
+    """The description of the current test case for hive."""
+    return f"Test: {request.node.name}"
```

Comment:
This will return the test ID, not the test case description as expected by Hive. Is this necessary? Perhaps you're intentionally overwriting this fixture?
https://github.com/ethereum/execution-specs/blob/15b5cfdfecdb66b525ca78379e9ad61bf54365bf/packages/testing/src/execution_testing/cli/pytest_commands/plugins/shared/execute_fill.py#L183-L205